src/source-pcap-file-directory-helper.h

```diff
@@ -51,6 +51,13 @@ typedef struct PcapFileDirectoryVars_
     PcapFileSharedVars *shared;
 } PcapFileDirectoryVars;
 
+/**
+ * Convert a timespec into milliseconds since epoch
+ * @param timespec
+ * @return time
+ */
+long AsEpochMillis(struct timespec *tm);
```

Comment:
I think this belongs in util-time.h (and the code in util-time.c). Also, I prefer explicit types, in this case probably uint64_t.

Reply:
If made a global func it would need to be renamed too. Something like TimespecAsMillis

Reply:
Added a PR (https://github.com/OISF/suricata/pull/3148) to add that as a global function to util-time.h.

Will create a new PR using that function when merged.